packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -24,18 +24,33 @@ export interface AutoUpdaterServiceInterface {
 }
 
 export default class AutoUpdaterService implements AutoUpdaterServiceInterface {
+	private window_: BrowserWindow;
+	private logger_: LoggerWrapper;
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	private initializedShim_: any;
```

Comment:
If `shim` is imported above using `import type shimType from '@joplin/lib/shim'`, something similar to the following might be a way to improve type safety:

```suggestion
	private initializedShim_: typeof shimType;
```